WTWSMS/decisions/realm_decisions.txt

```diff
@@ -2028,26 +2061,6 @@ decisions = {
 					value = off
 				}
 			}
-			
-			OR = {
-				ai = no
-				AND = {
-					trait = proud
-					prestige = 5000
-				}
-				AND = {
-					trait = ambitious
-					prestige = 5000
-				}
-				AND = {
-					realm_size = 70
-					any_demesne_title = {
-						count = 4
-						tier = DUKE
-					}
-				}
-				prestige = 15000
-			}
```

Comment:
> In this Mod, almost every starting kingdom and duchy is titular. Titles were constantly in flux. Besides, YOU are the one who dogs about "dynamic de jure" and "dynamic realms", but the code that is actually does it is restricted beyond any sane reason.


I am probably one of the largest proponents of dynamic dejure and dynamic realms, but not the one doing the code overhaul facilitating it which Enlil presides over. 

> Removing the prohibition of the AI from creating custom realms (and that is precisely what it is, as those numbers are flat-out insane) doesn't suddenly make it common! They still have to have the costs, they still have to have the clay or the duchies/kingdoms.

You want to remove all conditions. That is an extreme solution to a problem which doesn't exist. Yes the prestige requirements can be lowered, but not deleted. 

> I am willing to raise the amount of kingdoms needed for Empires, but the prestige requirements are insane. As high as they are now, you might as well put "AI = NO".

Raising the number of kingdoms doesn't make much sense as the current amount corresponds to dejure entities to a large extent.